src/router/index.ts

```diff
@@ -30,6 +30,12 @@ router.beforeEach((to, from, next) => {
     }
   }
 
+  // Currently only activate if the from route has a nrNumber query param
+  if (from.query.nrNumber && !to.query.nrNumber) {
+    // Carry over route params and allow the target route to overwrite any params with the same key
+    return next({ path: to.path, query: { ...from.query, ...to.query } })
```

Comment:
Looks OK but (if you haven't already) please test with the latest KC changes in master branch.

Reply:
@severinbeauvais you are right, I tried a fresh login and I lose query parameters when it passes through sign in. Works if you are already authenticated, will look into this.

Reply:
Updated and seems to be working now. I did the following:

- Added logic in the sign in component to pass on the query params.
- Added an authenticated flag to the store to trigger the fetching of data once the sign in process is completed
- Removed the v-if evaluatingPreconditions check on the <main> as it was preventing the sign in flow to fully execute